lib/questions.rb

```diff
@@ -1,22 +1,27 @@
 # keep only the elements that start with an a
 def select_elements_starting_with_a(array)
+  array.select{|string| string[0] == 'a'}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.